src/plugins/data/common/es_query/kuery/ast/kuery.peg

```diff
@@ -298,7 +299,7 @@ RangeOperator
   / '>' { return 'gt'; }
 
 Space "whitespace"
-  = [\ \t\r\n]
+  = [\ \t\r\n\u00A0]
```

Comment:
I think we can actually keep this here (here's an example): https://github.com/pegjs/pegjs/blob/b7b87ea8aeeaa1caf096e2da99fd95a971890ca1/examples/javascript.pegjs#L96

Reply:
Keeping